terratag_test.go

```diff
@@ -163,9 +202,9 @@ func getEntries(version string) []TestCase {
 	return testEntries
 }
 
-func cloneOutput(inputDirs []string) {
+func cloneOutput(inputDirs []string, terraformDir string) {
 	for _, inputDir := range inputDirs {
-		outputDir := strings.TrimSuffix(inputDir, "input") + "out"
+		outputDir := strings.TrimSuffix(inputDir, "input") + "out" + terraformDir
```

Comment:
just to prevent overriding on each other, each version clones to out + /terraformdir